packages/react/src/components/Dropdown/index.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { Dropdown } from '..';
+import { Item } from '..';
+
+const defaultProps = () => ({
+  id: `dropdown-${Math.random()}`,
+  onChange: action('onChange'),
+});
+
+storiesOf('Dropdowns/Dropdown', module)
+  .addDecorator(withKnobs)
+  .add('As Select component', () => (
+    <Dropdown {...defaultProps()} placeholder='Select...' isSelectable>
+      <Item type={Item.SIZES.MINI} optionValue='blueberry'>
+        Blueberry
+      </Item>
+      <Item type={Item.SIZES.MINI} optionValue='strawberry'>
+        Strawberry
+      </Item>
+    </Dropdown>
+  ))
+  .add('As Navigation', () => (
+    <Dropdown {...defaultProps()} placeholder='What do you wanna do?'>
+      <Item
+        type={Item.SIZES.NORMAL}
+        value='Take me there'
+        helper='Check what can really happend!'
+        onClick={action('onClick')}
+      >
+        Cars choice
+      </Item>
+      <Item
+        type={Item.SIZES.NORMAL}
```

Comment:
I'm pretty sure this is the default prop

Reply:
👍